websites/M/MKPC/presence.ts

```diff
@@ -0,0 +1,70 @@
+const presence = new Presence({
+  clientId: "812176837748457483"
+});
+
+var browsingStamp = Math.floor(Date.now() / 1000);
+
+var user: any;
+var title: any;
+var replace: any;
+var search: any;
+var post: any;
```

Comment:
no vars, no any type.

Reply:
Yes I know, will fix it XD, was planning to do var but website said no